ip-range-matcher/js/processor.js

```diff
@@ -0,0 +1,15 @@
+(function(){
+    'use strict';
+
+    var injector = angular.element("body").injector();
+    var ShakerProcessorsInfo = injector.get("ShakerProcessorsInfo");
+
+    ShakerProcessorsInfo.map["IPRangeMatcher"] = {
+        "description" : function(type, params){
+            if (!params.ipColumn) return null;
+            return "Matches IPs in column <strong>{0}</strong> against specified ranges".format(sanitize(params["ipColumn"]));
```

Comment:
Can add the number of ranges. Also, "matches" does not describe what's done with the match (action).